File: python/phone_number.py
def create_phone_number(n):
	num = []
	for i, c in enumerate(n):
		x = str(n[i])
		num.append(x)

	part_1 = "".join(num[0:3])
	part_2 = "".join(num[3:6])
	part_3 = "".join(num[6:])
	phone_number = ("(" + part_1 + ") " + part_2 + "-" + part_3)
	return phone_number

File: golang/phone_number.go
package kata

import (
		"strconv"
		"strings"
)

func CreatePhoneNumber(numbers [10]uint) string {
	digits := make([]string, 0)
	for _, number := range numbers {
		d := uint64(number)
		digit := strconv.FormatUint(d, 10)
		digits = append(digits, digit)
	}
	num := strings.Join(digits, "")
	phone := "(" + string(num[0:3]) + ") " + string(num[3:6]) + "-" + string(num[6:len(num)])
	return phone
}